hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -2,6 +2,9 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
+  // This is a configuration file for a simulation
+  dvsim_cfg_type:   "sim"
```

Comment:
We are calling this `flow` already.

Reply:
Good point. How about `dvsim_flow`?

Reply:
It is already known that DVSim caters to EDA tool flows, so perhaps just `flow` in this context is clear enough?

Reply:
Hmm, I think it's good to keep the "dvsim" name in there. Part of the point is to make sure that the hjson file that we're loading is definitely supposed to be loaded by dvsim.

Reply:
It is very unlikely any other HJson file which is not an input to DVSim will have the same key with the same list of valid values, so that check still applies. Besides, `{flow}` is used as substitution variable in a few places.

Reply:
Ok, if you feel strongly. I'll change it over.